src/views/Home/components/Milestones/index.scss

```diff
@@ -0,0 +1,223 @@
+@use "./stylesheets/responsiveness";
+@use "./stylesheets/typography";
+
+@mixin milestones-mobile {
+  .milestones-component {
+    &__header {
+      height: 100px;
+      width: 100%;
+      align-items: flex-start;
+    }
+    &__header-ellipse {
+      display: none;
+    }
+    &__header-circle {
+      width: 374px;
+      height: 87px;
+      border: 0;
+      justify-content: left;
+    }
+    &__header-text {
+      margin: 0;
+      padding: 0;
+      height: 67px;
+      width: 354px;
+      border-radius: 0%;
+      border-bottom-right-radius: 33.5px;
+      border-top-right-radius: 33.5px;
+      @include responsiveness.font($responsive: 4.1vw, $min: 25px, $max: 35px);
+    }
+    &__header-border {
+      position: relative;
+      bottom: 50px;
+      left: 210px;
+      display: block;
+    }
+    &__achievements {
+      .milestone {
+        &__image {
+          display: flex;
+          align-items: center;
+          justify-content: center;
+          min-width: 200px;
+        }
+        img {
+          width: 75%;
+          height: 60%;
+        }
+        &__box {
+          width: 180px;
+          justify-content: space-evenly;
+        }
+        &__stat {
+          line-height: 30px;
+          @include responsiveness.font(
+            $responsive: 4.1vw,
+            $min: 35px,
+            $max: 50px
+          );
+        }
+        &__name {
+          line-height: 25px;
+          @include responsiveness.font(
+            $responsive: 4.1vw,
+            $min: 21px,
+            $max: 28px
+          );
+        }
+      }
+    }
+  }
+}
+
+@mixin milestone-small {
+  .milestones-component__header-border {
+    position: relative;
+    display: block;
+  }
+  .milestones-component__header-circle {
+    min-width: 240px;
+    max-width: 320px;
+    width: 80%;
+    .milestones-component__header-text {
+      width: 304px;
+    }
+  }
+}
+
+.milestones-component {
+  margin: 14px 0 40px 0;
+  background: linear-gradient(
+    var(--color-light-pink),
+    var(--color-lavender),
+    var(--color-sky-blue)
+  ); //TODO: Name Color
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  flex-wrap: wrap;
+  &__header {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin-top: 3%;
+  }
+  &__header-text {
+    margin: 2% 0 1% 0;
+    height: 371px;
+    width: 371px;
+    background: var(--color-navy-blue);
+    border: 9px solid var(--color-navy-blue);
+    box-sizing: border-box;
+    box-shadow: 0px 4px 4px var(--color-gray);
+    border-radius: 50%;
+    font-family: Source Sans Pro;
```

Comment:
Fallback fonts in case `Source Sans Pro` is not usable? Also for `Lato` and `B612` below.